nncf/quantization/quantize_model.py

```diff
@@ -226,22 +227,44 @@ def quantize_with_accuracy_control(
 
 
 @api(canonical_alias="nncf.compress_weights")
-def compress_weights(model: TModel) -> TModel:
+def compress_weights(
+    model: TModel, mode=CompressWeightsMode.COMPRESSED_INT8, ratio: float = None, group_size: int = None
+) -> TModel:
     """
     Compress model weights.
 
     :param model: A model to be compressed.
+    :param mode: Defines a mode for weight compression.
+        COMPRESSED_INT8 stands for 8-bit integer quantization of all weights.
+        COMPRESSED_NF4 stands for a mixed-precision weights quantization to NF4 data type. The first and last layers
+        are always compressed to a backup precision which is uint8 by default. All others are quantized whether to NF4
+        or to a backup precision depending on criteria and the given ratio.
+    :param ratio: the ratio between baseline and backup precisions (e.g. 0.9 means 90% of layers quantized to NF4
+        and the rest to INT8).
+    :param group_size: number of weights (e.g. 128) in the channel dimension that share quantization parameters (scale).
+        The value -1 means no grouping. Defaults to -1.
```

Comment:
Please, align docstrings for `ratio` and `group_size` with implementation.

Reply:
thanks, aligned